lib/shared/bucketing-assembly-script/assembly/helpers/jsonHelpers.ts

```diff
@@ -46,6 +46,27 @@ export function isValidString(jsonObj: JSON.Obj, key: string, validStrings: stri
     return value
 }
 
+export function isUsableString(jsonObj: JSON.Obj, key: string, validStrings: string[]): string {
```

Comment:
the names of these methods are a bit weird. Unclear what the difference between `valid` and `usable` is, but I can't really think of a better distinction

Reply:
the other option I can think of is `isPassableString`, such that it won't error but just passthrough and return it?

Reply:
what about a "shouldThrow" parameter for the existing "valid" methods?

Reply:
yeah, that sounds better... let me update accordingly